api/openapi/content_stats_api.py

```diff
@@ -65,6 +53,18 @@ def get_stats(self, request: Request) -> Any:
 
         users_by_univ: dict[str, int] = {}
 
+        # A dict of pre-defined app types in the system.
+        # Undefined app types are dynamically added during processing.
+        # APP_REGISTRY is not used because its terminology is sligtly different.
+        apps_by_type: dict[str, int] = {
+            "customapp": 0,
+            "dashapp": 0,
+            "gradio": 0,
+            "shinyapp": 0,
+            "streamlit": 0,
+            "tissuumaps": 0,
+        }
```

Comment:
you need at the top update `from collections import defaultdict`

```suggestion
        apps_by_type: dict[str, int] = defaultdict(int)
        apps_by_type.update({
            "customapp": 0,
            "dashapp": 0,
            "gradio": 0,
            "shinyapp": 0,
            "streamlit": 0,
            "tissuumaps": 0,
        })
```

Reply:
Nice suggesstion, I will apply this.